miniirc_discord.py

```diff
@@ -86,7 +86,18 @@ def quote(self, *msg, force = None):
             self.debug('Changing online presence:', game)
             game = discord.Game(name = game)
             self._run(self._client.change_presence(game = game))
-
+        elif cmd in ('EMBED_NOTICE',):
```

Comment:
This should follow the [IRC](https://tools.ietf.org/html/rfc1459) [RFCs](https://tools.ietf.org/html/rfc2812) as closely as possible.

Reply:
### GAH